src/components/AccountVerificationProgress.tsx

```diff
@@ -0,0 +1,101 @@
+import confirmProfileFilled from '../assets/ConfirmProfileFilled.svg'
+import inActiveVerifyId from '../assets/InactiveVerifyId.svg'
+import verifyId from '../assets/VerifyID.svg'
+import verifyIdFilled from '../assets/VerifyIDFilled.svg'
+import inActiveSignPledge from '../assets/InactiveSignPledge.svg'
+import activeSignPledge from '../assets/SignPledge.svg'
+import { ValidationWizardStep } from './ProfileValidation'
+
+export const PlaneToRocket = () => {
+    return(
+        <div>
+            <img src={PaperAirPlane} alt ='paper airplane'/>
+            <img src={Arrow} alt ='arrow' style={{margin:'0 16px'}}/> 
+            <img src={RocketShip} alt='rocket ship'/>
+        </div>
+    )
+}
+
+interface VerificationProgressProps {
+    currentProgress: number
+    imgHeight: number
+    profileSrc: string
+    verifySrc: string
+    signPledgeSrc: string
+    profileFont: string
+    identityFont: string 
+    signFont: string
+}
+
+export const VerificationProgress: React.FC<VerificationProgressProps> = (props: VerificationProgressProps) => {
+    return (
+        <div className='progress-container'>
+            <div className='progress-element progress-border' style={{width:'100%',top:props.imgHeight / 2}}></div>
+            <div className='progress-element progress-border'  style={{width:`${props.currentProgress}%`,top:props.imgHeight / 2,color:'#4DB3B9'}}></div>
+            <img className='progress-element' src={props.profileSrc} alt= 'profile' style={{left:0}}/> 
+            <img className='progress-element verify-identity' src={props.verifySrc} alt='verifyId'/> 
+            <img className='progress-element' src={props.signPledgeSrc} alt='signPledgeSrc' style={{right:0}}/>
+            <div className={`progress-element ${props.profileFont}`} style={{left:-props.imgHeight, top:props.imgHeight*1.5}}>1. Confirm Profile</div>
+            <div className={`progress-element verify-identity ${props.identityFont}`} style={{ top:props.imgHeight*1.5}}>2. Verify Identity</div>
+            <div className={`progress-element ${props.signFont}`} style={{right:-props.imgHeight, top:props.imgHeight*1.5}}>3.Sign Pledge</div>
+        </div>
+    ) 
+}
+
+interface AccountVerificationProgressProps {
+    step: ValidationWizardStep
+}
+
+export const AccountVerificationProgess: React.FC<AccountVerificationProgressProps> = ({step}) => {
+
+    return(
+        <div className='progress-indicator'>
+            <div style ={{textAlign:'center'}}>
+                <PlaneToRocket />
+                <span>Account Verification</span>
+            </div>
+
+            {step === ValidationWizardStep.PROFILE_INFO && 
+                <VerificationProgress 
+                currentProgress = {15} 
+                imgHeight = {40} 
+                profileSrc={confirmProfile}
+                verifySrc={inActiveVerifyId}
+                signPledgeSrc={inActiveSignPledge}
+                profileFont='current-font'
```

Comment:
This is for setting the class name to change the font based on the step. Was wondering if there was a way of changing it dynamically other than passing props.